Makefile

```diff
@@ -295,7 +295,7 @@ test/products:
 	delorean pipeline product-tests --test-config ./test-containers.yaml --output $(TEST_RESULTS_DIR) --namespace test-products
 
 .PHONY: cluster/deploy
-cluster/deploy: kustomize cluster/cleanup cluster/cleanup/crds cluster/prepare/crd cluster/prepare cluster/prepare/rbac/dedicated-admins deploy/integreatly-rhmi-cr.yml
+cluster/deploy: kustomize cluster/cleanup cluster/cleanup/crds cluster/prepare/crd cluster/prepare cluster/prepare/rbac/dedicated-admins deploy/integreatly-rhmi-cr.yml install-oo
```

Comment:
so this command is used by both: e2e command and pipelines (using directly cluster/deploy)
In e2e, we don't want the install-oo, in pipelines we do. 

Maybe another command: make cluster/deploy/e2e - which would not install OO? WDYT?

Reply:
Done in commit 3, hope this is what you asked. Thank you!